src/Microsoft.TemplateEngine.Cli/HelpAndUsage/HelpForTemplateResolution.cs

```diff
@@ -399,6 +412,11 @@ public static CreationResultStatus HandleParseError(INewCommandInput commandInpu
             return CreationResultStatus.InvalidParamValues;
         }
 
+        public static string GetTemplateHelpCommand(INewCommandInput commandInput, ITemplateInfo template)
```

Comment:
nit: No need for this method to be public as it's called only from this class.

Reply:
Changed to internal, spotted that I can also use this method in TemplateUsageHelp.ShowInvocationExamples()